src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/OveruseEstimator.java

```diff
@@ -99,6 +100,8 @@ public OveruseEstimator(OverUseDetectorOptions options)
         varNoise = options.initialVarNoise;
         E = clone(options.initialE);
         processNoise = options.initialProcessNoise.clone();
+        //init with max value to simplify updateMinFramePeriod
```

Comment:
I don't like such formatting of comments. Make it a proper sentence in written English please.